src/flashcart/sc64/sc64.c

```diff
@@ -214,6 +214,39 @@ static flashcart_error_t sc64_load_rom (char *rom_path, bool byte_swap) {
     return FLASHCART_OK;
 }
 
+static flashcart_error_t sc64_load_emulator_rom (char *rom_path) {
```

Comment:
I wonder if we can generalize this function as "load file at ROM offset", that might be beneficial in the future if other emulators choose to use other offset than `0x200000`.

Reply:
I would agree. A function like that would definitely be easier to handle. Perhaps `load_rom (char *rom_path, uint32_t start_offset_address)` ?

Reply:
I don't think changing `load_rom` is correct because it has optimizations for straight up loading default N64 ROM with all quirks that flashcart might require. I think separate function `load_file (char *file_path, uint32_t start_offset_address)` would be better.